src/StackExchange.Redis/PhysicalConnection.cs

```diff
@@ -720,8 +721,13 @@ internal void GetStormLog(StringBuilder sb)
             }
         }
 
-        internal void OnBridgeHeartbeat()
+        /// <summary>
+        /// Runs on every heartbeat for a bridge, timing out any commands that are overdue and returning an integer of how many we timed out.
+        /// </summary>
+        /// <returns>How many commands were overdue and threw timeout exceptions.</returns>
+        internal int OnBridgeHeartbeat()
```

Comment:
We only really _need_ this to return a `bool` indicating if _any_ commands were timed out. Or will the `int` be useful in some scenarios?

Reply:
I meant to include this in the exception...woops. Fixed!